README.md

```diff
@@ -140,6 +141,22 @@ In Trace Analytics you can visualize the following tables.
 
 In Insights you can see the summary table for Insights. Clicking the InsightId will take you to AWS console.
 
+### Service map
+
+[TODO: image of the map in grafana]
+
+Service map shows the same data as a service map inside X-ray console. It needs Grafana v7.4+ to use it's Node graph visualization otherwise it will show the data in a simple table.
```

Comment:
I am confused. What is "Service map" in this context? Is it something in the X-ray data source in Grafana?

Reply:
"Service map" here means data that you get when run a "service map" query in the X-ray data source.

Reply:
Is "Service map" used in the UI, either for the data source or the plugin? If so, please match the UI capitalization.

If not, then will it be clear to users what you mean by "service map"?

Reply:
Yeah wonder about that, we have Service Map in the query type:
![Screenshot from 2021-01-25 14-25-06](https://user-images.githubusercontent.com/1014802/105711736-3f8da280-5f19-11eb-8e71-54d5f8c12ee8.png)

So the first service means the query type inside the plugin, the second one means service map inside the X-ray console. Does it make sense in that case to change to `Service Map shows the same data as a service map inside X-ray console` do differentiate that?

Reply:
Yes, that would be perfect.